lib/internal/perf/performance_entry.js

```diff
@@ -92,9 +94,11 @@ function initPerformanceEntry(entry, name, type, start, duration) {
 }
 
 function createPerformanceEntry(name, type, start, duration) {
-  return ReflectConstruct(function PerformanceEntry() {
-    initPerformanceEntry(this, name, type, start, duration);
-  }, [], PerformanceEntry);
+  const entry = new PerformanceEntry(kSkipThrow);
+
+  initPerformanceEntry(entry, name, type, start, duration);
```

Comment:
As mentioned by Qard in the other PR you can just move the initPerformanceEntry logic to the constructor - otherwise it's needless indirection

Reply:
Done!